apps/mobile/components/SSWordInput.tsx

```diff
@@ -10,42 +10,53 @@ type SSWordInputProps = {
   invalid?: boolean
   position: number
   editable?: boolean
+  index: number
 } & React.ComponentPropsWithoutRef<typeof TextInput>
 
-export default function SSWordInput({
-  value,
-  invalid,
-  position,
-  editable = true,
-  style,
-  ...props
-}: SSWordInputProps) {
-  const textInputStyle = useMemo(() => {
-    return StyleSheet.compose(
-      {
-        ...styles.textInputBase,
-        ...(invalid ? styles.textInputInvalid : {})
-      },
-      style
+const SSWordInput = forwardRef(
```

Comment:
can we maintain the `function` keyword instead of arrow function, same as we doing for other components?

Reply:
`function SSWordInput`

`export default forwardRef(SSWordInput)`

this might work?